src/ImageSharp/Advanced/AdvancedImageExtensions.cs

```diff
@@ -148,7 +148,7 @@ public static Memory<TPixel> DangerousGetPixelRowMemory<TPixel>(this ImageFrame<
         Guard.MustBeGreaterThanOrEqualTo(rowIndex, 0, nameof(rowIndex));
         Guard.MustBeLessThan(rowIndex, source.Height, nameof(rowIndex));
 
-        return source.PixelBuffer.GetSafeRowMemory(rowIndex);
+        return source.PixelBuffer!.GetSafeRowMemory(rowIndex);
```

Comment:
@JimBobSquarePants Should I look for all places where I used ! and add an ArgumentNullException.ThrowIfNull?